test/functional/rawtransactions.py

```diff
@@ -263,7 +302,7 @@ def run_test(self):
         encrawtx = "01000000010000000000000072c1a6a246ae63f74f931e8365e15a089c68d61900000000000000000000ffffffff0100e1f505000000000000000000"
         decrawtx = self.nodes[0].decoderawtransaction(encrawtx, False) # decode as non-witness transaction
         assert_equal(decrawtx['vout'][0]['value'], Decimal('1.00000000'))
-        
+
```

Comment:
👀

Reply:
Removing trailing white space should be fine now, since adding trailing white space is forbidden. So we converge to a single "code style".